contents/handbook/small-teams/marketing/objectives.mdx

```diff
@@ -1,18 +1,36 @@
 * **Objective**:
-    * Expand PostHog’s top of funnel awareness across all our core _individual_ products - product analytics, session recording, feature flags, and A/B testing.
+    * Increase word-of-mouth growth by:
+      1. being an authority and vital resource on building successful products.
+      2. creating a world-class community experience.
 
-* **Key Results:**
-    * Increase [overall weekly website traffic](https://app.posthog.com/insights/hg6kfoIG) to 27k unique visitors per week (filtering out viral spikes).
-    * Increase [‘first user in org’ signups](https://app.posthog.com/insights/EwWz7QuV) to 600 per week without decreasing the percentage that are ICPs (20-25%).
-    * Make Tutorials our [3rd most visited section](https://app.posthog.com/insights/n8b9SHgd) (after Docs and Blogs)
-    * Increase number of [subscribers](https://newsletter.posthog.com/publish/subscribers) to Product for Engineers to 1,500
-    * Get [200 orgs](https://docs.google.com/spreadsheets/d/1ryDGzXcG0OSH-GoX-zlI2CKwNu-6Zivi7EM32WLdids/edit#gid=0) to sign up to the startup deal
-    * Increase [28 day YouTube watch time](https://studio.youtube.com/channel/UCn4mJ4kK5KVSvozJre645LA?c=UCn4mJ4kK5KVSvozJre645LA) to 227 hours
+* **Goals:**
```

Comment:
Remember when we said we were spread too thin with our goals last quarter? We have seemingly the same amount of goals, plus "business as usual".

Reply:
I think the difference, if people agree with this approach, is these goals have owners, whereas before they were defuse and non-specific.

On reflection, though, I think:

1. We need a BAU goal, such as "Continue to grow SEO traffic and improve documentation".  
2. We should trim the list a little, either by dropping some or combining them – i.e. PostHog for Startups IS our community plan.

Open to ideas here as I think it's important to get this right rather than waving through stuff we're not 100% behind.

Reply:
I think the community should be the startups program. I think we can drop content discovery to being a Web & Docs function. That brings us to three without losing anything IMHO. 

Regarding ownership, happy to have single owners (and be the single owner for startups) but I personally also think we work well when collaborating. New goal: SYNERGY.

```
    * Nail Twitter and content distribution (**Owner:** Charles)
    * Create and execute a plan to make PostHog for Startups the default community for technical founders (**Owner:** Joe & Lior)
    * Scale the newsletter to thousands of engaged subscribers (**Owner:** Andy & Ian)
```

That said, I care less about planning the work and more about doing the work here. I feel we have a consensus direction, so I'm happy to wave things through a bit now.

Reply:
Need to add Lottie's as well and the BAU one, but I think the above makes sense. Thoughts @jamesefhawkins @Lior539?

Reply:
I dont have strong opinions on this. Happy to go with @joethreepwood 's suggestion, or keep as is.

> Remember when we said we were spread too thin with our goals last quarter? 

The new "goals" read to me more like "projects", so this is why I don't really have an issue with keeping it as is. I think it's fine for us to have many projects 

I dont think of them as "goals". The "mission" is the goal

Reply:
I'm fine with dropping internal discovery and just working with website & docs where needed. I think a lot of that will be driven by them.